cdap-integration-test/src/main/java/co/cask/cdap/test/IntegrationTestBase.java

```diff
@@ -119,7 +118,7 @@ public Boolean call() throws Exception {
           throw Throwables.propagate(ex);
```

Comment:
changed it to catch ``IOException``. However, still unsure if we should throw a ``ServiceUnavailableException`` in this case. Will think about it.